blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionServer.java

```diff
@@ -72,10 +75,13 @@
     public HttpMetricsIngestionServer(ScheduleContext context, IMetricsWriter writer) {
         this.httpIngestPort = Configuration.getInstance().getIntegerProperty(HttpConfig.HTTP_INGESTION_PORT);
         this.httpIngestHost = Configuration.getInstance().getStringProperty(HttpConfig.HTTP_INGESTION_HOST);
-        int acceptThreads = Configuration.getInstance().getIntegerProperty(HttpConfig.MAX_WRITE_ACCEPT_THREADS);
-        int workerThreads = Configuration.getInstance().getIntegerProperty(HttpConfig.MAX_WRITE_WORKER_THREADS);
         this.timeout = DEFAULT_TIMEOUT; //TODO: make configurable
         this.processor = new Processor(context, writer, timeout);
+    }
+
+    public void startServer() {
```

Comment:
@chinmay-gupte TL;DR There was no way to plug another EventsIngestionHandler in the HttpMetricsIngestionServer, this code makes it possible now. 

Before this commit, in HttpIngestionService class the metrics ingestion server was started with this piece of code

```
server = new HttpMetricsIngestionServer(context, writer)
``` 

It was impossible to plug another server (with more of it's components plugged) for testing.  Therefore, I extracted the start code of the server to a separate startServer method. So in HttpIngestionService it looks like this.

```
  server = getServer().startServer(); 
```  

I can use the setServer to plug the server for testing and then explicitly start it. Make sense?